tests/Microsoft.Identity.Test.Unit.net45/CacheTests/UnifiedSchemaValidationTests.cs

```diff
@@ -746,15 +714,13 @@ public void TestSchemaComplianceForIDToken_WhenMSSTSResponse_WithAADAccountAndFo
             Assert.AreEqual(2003, key.iOSType);
         }
 
+        // TODO: We have not, on .NET, implemented saving two different fresh tokens in FOCI case (one FRT and one MRT).  So this test won't succeed.
+        // Marking it ignored now until we get through other PRs active for this to get resolved.
```

Comment:
@SomkaPe is driving an redesign of FOCI that states that an FRT will be saved as an FRT alone. IMO the schema tests will need to be updated.